packages/alfa-rules/src/common/predicate/is-visible.ts

```diff
@@ -42,7 +43,9 @@ function isInvisible(device: Device, context?: Context): Predicate<Node> {
       .get(node, () =>
         test(
           or(
-            not(isRendered(device, context)),
+            not(
+              isRendered(device, context)
+              ),
```

Comment:
```suggestion
            not(isRendered(device, context)),
```
No reason to make it multiline 🤔